core/src/SYCL/Kokkos_SYCL_Instance.cpp

```diff
@@ -150,8 +150,13 @@ void SYCLInternal::initialize(const sycl::queue& q) {
 
     m_maxShmemPerBlock =
         d.template get_info<sycl::info::device::local_mem_size>();
-    m_indirectKernelMem.reset(*m_queue, m_instance_id);
+
     m_indirectReducerMem.reset(*m_queue, m_instance_id);
+    for (auto& usm_mem : m_indirectKernelMem) {
+      usm_mem.reset(*m_queue, m_instance_id);
+      usm_mem.reserve(0x1440);  // TODO joe: arbitrary (probably too large) size
```

Comment:
Please use decimal or add a comment saying what value this is as most of us are not fluent in hexadecimal.